### ISSUE TITLE
Type.createIntance is marked as error

### ISSUE THREAD

**Issue by CrazyFlasher** [HaxeFoundation/intellij-haxe#981]
```haxe
class Test {
    static function main() {
        trace("Haxe is great!");
        
        var instance:Dynamic = Type.createInstance(Test, []);
    }
}
```

**Comment by EricBishton** [comment]
Fixed with PR#995, https://github.com/HaxeFoundation/intellij-haxe/commit/c4cc7763b12fde6b200fd2c5fd59efb3366de527

**Comment by CanIGetaPR** [comment]
Works correctly on plugin version 1.3.1 this can be closed I believe.
